src/main/ui/app/sass/addonindex.scss

```diff
@@ -26,14 +26,32 @@
   float: right;
 
 }
+.maintable{
+margin-bottom:0;
+}
+
+
+.titletype{
```

Comment:
Can't we use a standard bootstrap class for this stuff? I'm surprised we need to this much customization.